macro/vignettes/fluxmovement.Rmd

```diff
@@ -0,0 +1,146 @@
+---
```

Comment:
This vignette is self-contained, which is very nice. It's a good bit of proof.
We should make running the vignettes part of the testing b/c this is a unit test.

Reply:
I can just put this code into the unit tests rather than doing that. I think I have done that already for the simple trip module now that I look at it again.